website/source/docs/auth/kubernetes.html.md

```diff
@@ -130,6 +130,7 @@ versions prior should ensure the Kubernetes API server is started with with this
 setting. Otherwise deleted tokens in Kubernetes will not be properly revoked and
 will be able to authenticate to this backend. 
 
+### RBAC Configuration
```

Comment:
I'm sure most Kube admin know what RBAC means but I don't.  Maybe spell out RBAC in the paragraph below.